templates/conf/tfstate-backend/terraform.tfvars

```diff
@@ -0,0 +1 @@
+aws_region = "${aws_region}"
```

Comment:
The variable is called `region` so it has to be `region = "${aws_region}"`. The docker images have to be recompiled for this, of course.

Reply:
Change made. This makes sense.

This should in theory work then if the variable is called `region` and not `aws_region` as I thought.

Reply:
Yep, I verified with my infrastructure.

Reply:
Thanks for testing!